include/curl/curl.h

```diff
@@ -3238,4 +3238,5 @@ CURL_EXTERN CURLcode curl_easy_pause(CURL *handle, int bitmask);
 #endif /* __STDC__ >= 1 */
 #endif /* gcc >= 4.3 && !__cplusplus && !CURL_DISABLE_TYPECHECK */
 
+extern long curl_to_ares_timeout_ms;
```

Comment:
A hard no here. You cannot introduce global variables - anywhere - in curl or libcurl.

Reply:
Hi @bagder ,
I want use this timeout_ms variable in Curl_resolver_init() function but not able to use. Can you please point me how can i access this variable in this Curl_resolver_init() function.

Thanks

Reply:
Options passed to the library are done with `curl_easy_setopt`. There are 300+ options there already. You can check existing timeout options for how they are set form the tool and used by the library.

That said, I don't think users want to set this timeout separately.